### DISCUSSION TITLE
How to Set Focus to a specifc Cell in Slick grid?

### DISCUSSION THREAD

**Discussion by ibrahimGit123** [serenity-is/Serenity#5921] [Help] [answered]
I am trying to focus to a specifccell in Slickgrid (with inline editing), but I am unable to do so. 

See this thread for reference:
https://stackoverflow.com/questions/12842946/select-focus-cell-using-slickgrid

I am calling these methods but no luck.

**Comment by VictorTomaili** [comment, marked as answer]
if you have specific data attribute on this cell (probably you need to use formatter for that) you can try to find this cell with a selector.

**Reply by ibrahimGit123** [reply]
What the code to do this?

**Reply by VictorTomaili** [reply]
https://docs.serenity.is/docs/howto/how_to_use_a_slickgrid_formatter

You can use your row data (for example row id field) to implement custom attribute data on formatter (Like `<div data-row-id="123"></div>`) then you can select this cell with jquery and css selector.
